nuxeo-services/nuxeo-collections/src/main/java/org/nuxeo/ecm/collections/core/CollectionManagerImpl.java

```diff
@@ -164,14 +164,13 @@ protected void checkCanCollectInCollection(final DocumentModel collection, final
     protected DocumentModel createCollection(final String newTitle, final String newDescription,
             final DocumentModel context, final CoreSession session) {
         DocumentModel defaultCollections = getUserDefaultCollections(session);
+        PathSegmentService pss = Framework.getService(PathSegmentService.class);
+        String pssTitle = pss.generatePathSegment(newTitle);
         Map<String, Object> options = new HashMap<>();
         options.put(CoreEventConstants.PARENT_PATH, defaultCollections.getPath().toString());
-        options.put(CoreEventConstants.DESTINATION_NAME, newTitle);
-        options.put(CoreEventConstants.DESTINATION_NAME, newTitle);
+        options.put(CoreEventConstants.DESTINATION_NAME, pssTitle);
```

Comment:
Is this change related to https://jira.nuxeo.com/browse/NXP-26903?
In wich case it would deserve a unit test please :pray:

Reply:
Actually without this change the unit est ```org.nuxeo.ecm.collections.core.test.CollectionAddRemoveTest.testAddCollectionWithInvalidName()``` will fail

Reply:
OK, then, with your change, the following line
```
DocumentModel newCollection = session.createDocumentModel(CollectionConstants.COLLECTION_TYPE, options);
```
will call `AbstractSession#createDocumentModelFromTypeName` that will call `DocumentModel#setPathInfo` (that's why you need to fix the `CoreEventConstants.DESTINATION_NAME` option first.
In this case, you can also remove
```
newCollection.setPathInfo(defaultCollections.getPath().toString(), pssTitle);
```
from below.